packages/excalidraw/tests/data/restore.test.ts

```diff
@@ -290,12 +312,11 @@ describe("restoreElements", () => {
     expect(restoredElements).toEqual([
       expect.objectContaining({
         id: rectangle.id,
-        version: rectangle_modified.version + 1,
+        version: rectangle_modified.version + 2,
```

Comment:
not particularly in favour of the double version bump, but I expect we could live some time without it;

ideally it should be solved by generically batching mutations (with possible rollback)

Reply:
Related to #7359